hexa/workspaces/models.py

```diff
@@ -58,6 +59,7 @@ class Workspace(Base):
     description = models.TextField(blank=True)
     members = models.ManyToManyField(User, through="WorkspaceMembership")
     countries = CountryField(multiple=True, blank=True)
+    database = models.ForeignKey(Database, null=True, on_delete=models.SET_NULL)
```

Comment:
Since we haven't defined the model yet, I think it's better to not have this column. @pvanliefland WDYT?

Reply:
@qgerome @cheikhgwane I agree - not needed for now

Reply:
Okay !